django/contrib/auth/views.py

```diff
@@ -200,6 +201,9 @@ def password_reset(request, is_admin_site=False,
                     RemovedInDjango110Warning, 3
                 )
                 opts = dict(opts, domain_override=request.get_host())
+
+            opts.update({'extra_email_context': extra_email_context})
```

Comment:
As I tried to comment earlier, there is no need to use `opts.update()`. Just put this key value pair in the declaration of the `opts` dictionary.